pkgs/by-name/pr/present-cli/package.nix

```diff
@@ -6,17 +6,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "present";
-  version = "0.2.2";
+  version = "0.2.3";
 
   src = fetchFromGitHub {
     owner = "terror";
     repo = pname;
-    rev = version;
-    sha256 = "+kCHe84ikdCLd7j5YwP2j3xz+XTzzo/kLy+b9YUFDnI=";
+    rev = "43c10253dc31038614eba5824588dbf2716212d6A";
```

Comment:
Why not using the tag?

Reply:
The tag got mis-assigned one commit before the version bump in Cargo.toml so this references the commit with the actual version bump included.